DSharpPlus/Entities/DiscordChannel.cs

```diff
@@ -279,6 +283,31 @@ public Task<DiscordMessage> SendMultipleFilesAsync(Dictionary<string, Stream> fi
             return this.Discord.ApiClient.UploadFilesAsync(this.Id, files, content, tts, embed, mentions);
         }
 
+        /// <summary>
+        /// Sends a Message.
+        /// </summary>
+        /// <param name="content">The content to send.</param>
+        /// <returns></returns>
+        public Task<DiscordMessage> SendMessageAsync(string content)
+        {
+            if (this.Type != ChannelType.Text && this.Type != ChannelType.Private && this.Type != ChannelType.Group && this.Type != ChannelType.News)
+                throw new ArgumentException("Cannot send a text message to a non-text channel.");
+
+            return this.Discord.ApiClient.CreateMessageAsync(this.Id, content, null, null, null);
+        }
+
+        /// <summary>
+        /// Sends a Message
```

Comment:
Please reuse the same documentation as older methods.

Reply:
This is completed.